earthpy/io.py

```diff
@@ -267,7 +274,8 @@ def _download(self, url, path, kind, replace, verbose):
             return path
 
         if verbose is True:
```

Comment:
This doesn't print the log statement regardless of how we configure things. it's almost as if setting the level is not working.